### ISSUE TITLE
[Bug]: Card Today Activity Show Twice

### ISSUE THREAD

**Issue by Cedric921** [ever-co/ever-teams#2575]
We should not see today's activity twice on the Card,
If the user expands the Card (Near of the assigned Member), we should no longer have the possibility of expand on the other buttons (Total Worked Today or Task Info)

https://github.com/ever-co/ever-teams/assets/83704005/3fcfb272-b022-4b65-a087-00a1e7c4fe9e

**Comment by evereq** [comment]
Yes, if one block is expanded and the user clicks another "Expand" button, it should replace the currently expanded block content with another expanded block content (replace CONTENT inside if it's possible... if not possible or too hard, we can close one expanded and open another expanded etc). Is that correct @Cedric921 ? 
CC: @GedeonTS

**Comment by Cedric921** [comment]
> Yes, if one block is expanded and the user clicks another "Expand" button, it should replace the currently expanded block content with another expanded block content (replace CONTENT inside if it's possible... if not possible or too hard, we can close one expanded and open another expanded etc). Is that correct @Cedric921 ?
> CC: @GedeonTS

Correct Sir @evereq
